### ISSUE TITLE
Documentation issues in V4.4.0

### ISSUE THREAD

**Issue by heathhayward** [labordynamicsinstitute/qwi_schemas#106]
- Supercedes: https://labordynamicsinstitute.github.io/qwi_schemas/formats/V4.4.0/lehd_public_use_schema.html#_supersedes (also in the naming convention supercedes section: https://labordynamicsinstitute.github.io/qwi_schemas/formats/V4.4.0/lehd_csv_naming.html#_supersedes) should be changed to "V4.3.1".  This hadn't been updated in awhile.

- Geographic Levels table: https://labordynamicsinstitute.github.io/qwi_schemas/formats/V4.4.0/lehd_public_use_schema.html#geo_level isn't displaying correctly. The "Source" column header is appearing as a value underneath "geo_level".  The spacing of the columns could be better as well to allow more space for the long descriptions.

- State Postal Codes table: https://labordynamicsinstitute.github.io/qwi_schemas/formats/V4.4.0/lehd_public_use_schema.html#_state_postal_codes also has wrapping issues.  In V4.3.1, the "labels" column in "label_stusps.csv" was not displayed, but it looks like it's trying to display here.

- In the Aggregation level: https://labordynamicsinstitute.github.io/qwi_schemas/formats/V4.4.0/lehd_public_use_schema.html#_aggregation_level section, the 2nd table is a mess and seems to have the above issue of the csv having more columns than the display has allocated.

- In the Version Metadata: https://labordynamicsinstitute.github.io/qwi_schemas/formats/V4.4.0/lehd_public_use_schema.html#_version_metadata_for_qwi_and_j2j_files_version_txt section, the 2nd embedded version file (J2J for DE R2018Q2) isn't showing the actual contents of the table. It's showing : "<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN"> ..." instead.

- In the Filenaming Schema table: https://labordynamicsinstitute.github.io/qwi_schemas/formats/V4.4.0/lehd_csv_naming.html#_basic_filename_schema, there are three "\" in the 2nd row of the Type column and one in the 4th row (although this one appears in V4.3.1 as well) that look to be escaping underscores, but they should not appear in this table.

**Comment by larsvilhuber** [comment]
@jodyhoonstarr are you working on this? Please let me know.

**Comment by jhoonstarrNLT** [comment]
@larsvilhuber Yeah, I have a local docker setup with this repo and the asciidoctor image. I'll start working through these issues.

**Comment by jodyhoonstarr** [comment]
@larsvilhuber Yeah, I have a local docker setup with this repo and the asciidoctor image. I'll start working through these issues.

**Comment by jodyhoonstarr** [comment]
@larsvilhuber Any thoughts on getting the long descriptions from the geographic levels csv formatted correctly? I can't get asciidoctor to recognize the column widths (i.e. cols="^1,<3,<4,<2,<1") format using either relative values or percentages.

**Comment by larsvilhuber** [comment]
commit your branch back to github, and I can take a stab at it.

--
Lars Vilhuber, Economist
Cornell University, Executive Director, Labor Dynamics Institute
  and ILR School - Department of Economics
American Economic Association - Data Editor
Journal of Privacy and Confidentiality - Managing Editor

e: lars.vilhuber@cornell.edu
p: +1.607-330-5743
v: https://cornell.zoom.us/my/larsvilhuber
w: http://lars.vilhuber.com/ <http://lars.vilhuber.com/>

Assistant: ldi@cornell.edu | +1.607-255-2744
________________________________

**Comment by jodyhoonstarr** [comment]
I actually think this may be related to https://github.com/asciidoctor/asciidoctor/issues/2186. I may just surrender on styling and get the content right for now.

**Comment by larsvilhuber** [comment]
This is where our mixture of scripts and asciidoc can help. We can use scripts to modify the content for display only. Point me to the exact problem, and I can offer a solution.

**Comment by jodyhoonstarr** [comment]
Ahh, got it - will just filter bits of the csv in bash. Thanks, I'll ping you if I run into other issues.

**Comment by larsvilhuber** [comment]
Just remember to pipe into a tmp[N].csv, not modifying the original CSV... Wouldn't want to mess up the schema...

--
Lars Vilhuber, Economist
Cornell University, Executive Director, Labor Dynamics Institute
  and ILR School - Department of Economics
American Economic Association - Data Editor
Journal of Privacy and Confidentiality - Managing Editor

e: lars.vilhuber@cornell.edu
p: +1.607-330-5743
v: https://cornell.zoom.us/my/larsvilhuber
w: http://lars.vilhuber.com/ <http://lars.vilhuber.com/>

Assistant: ldi@cornell.edu | +1.607-255-2744
________________________________

**Comment by jodyhoonstarr** [comment]
Nearly done the top few. Adding a few more issues that have been brought up. 

- Too much white space in the tables (might be fixed with your work on table formatting)

- Replace all instances of the text "http" with "https". I think all the hyperlinks are fine (since they don't specify the http part of the URL), so it's just the plan text that needs to change.

- In http://lehd-app1.dev.ces.census.gov/data/schema/V4.4.0/lehd_shapefiles.html#_scope second paragraph, make "QWI" and "J2J" text hyperlinks rather than displaying the URLs in brackets after.

- Remove extra closing bracket from "[type]]" in http://lehd-app1.dev.ces.census.gov/data/schema/V4.4.0/lehd_shapefiles.html#_basic_naming_schema.

-  In http://lehd-app1.dev.ces.census.gov/data/schema/V4.4.0/lehd_shapefiles.html#_geography, make the "The valid codes correspond to..." sentence it's own line (so that the hyperlink is on it's own line.  Also delete "and label_geography_metro.csv" from the sentence since the "B" geo_levels are now included in the label_geography.csv file.

- In http://lehd-app1.dev.ces.census.gov/data/schema/V4.4.0/lehd_shapefiles.html#_name, add "(label_geography.csv) as it's own line at the beginning of this section (as done in other sections). Also delete "and label_geography_metro.csv" from the sentence since the "B" geo_levels are now included in the label_geography.csv file.

**Comment by heathhayward** [comment]
Found a couple more issues:

- In http://lehd-app1.dev.ces.census.gov/data/schema/V4.4.0/lehd_public_use_schema.html#geo_level, the last sentence should read:  "The 2018 vintage of [Census TIGER/Line geography](https://www.census.gov/geo/maps-data/data/tiger-line.html) is used for all tabulations as of the R2019Q1 release."

- In http://lehd-app1.dev.ces.census.gov/data/schema/V4.4.0/lehd_shapefiles.html#_sources, replace all "2017" text with "20018" and update the URLs to go to "../TIGER2018/..."

- For the above change, does http://lehd-app1.dev.ces.census.gov/data/schema/V4.4.0/lehd_shapefiles.html#changes need to be updated?

**Comment by larsvilhuber** [comment]
Yes, that should be updated for the changes file. Are there also changes to some URLs that need to be updated?

--
Lars Vilhuber
on mobile device

________________________________
